pod2daemon/Makefile

```diff
@@ -78,7 +78,7 @@ $(BINDIR)/csi-driver-%: $(SRC_FILES)
 	$(call build_binary, csidriver/main.go, $@)
 
 UPSTREAM_REGISTRAR_PROJECT ?= kubernetes-csi/$(REGISTRAR_IMAGE)
-UPSTREAM_REGISTRAR_TAG     ?= a498c31b88cd31f8ea7bafd0d715a75f9939fa1f
+UPSTREAM_REGISTRAR_TAG     ?= 9b41324b20f609bce4dda9e50cc484c4ef4f5949 #https://github.com/kubernetes-csi/node-driver-registrar/releases/tag/v2.13.0
```

Comment:
I think we can just mention the version directly (instead of the hash) and it will work. We only start using hash since upstream didn't have all the CVE fixes that we wanted.